src/assets/js/script.js

```diff
@@ -0,0 +1,37 @@
+const modal = document.querySelector(".modal");
+const modalImg = document.querySelector(".modal__content");
+
+document.querySelector(".modal").addEventListener("click", () => {
+  modal.classList.remove("showModal");
+});
+
+document
+  .querySelector(".work__content")
+  .addEventListener("click", function (event) {
+    let targetImg = event.target;
+
+    if (targetImg.tagName != "IMG") return;
+
+    modal.classList.add("showModal");
+    modalImg.src = targetImg.src;
```

Comment:
Clear attribute value when you hide the modal.

Reply:
Thanks, fixed it.
```suggestion
modal.addEventListener("click", () => {
  modal.classList.remove("showModal");
  modalImg.setAttribute("src", "");
});

document
  .querySelector(".work__content")
  .addEventListener("click", function (event) {
    let targetImg = event.target;

    if (targetImg.tagName !== "IMG") return;

    modal.classList.add("showModal");
    modalImg.setAttribute("src", targetImg.src);
  });
});
```